vendor/github.com/docker/docker/client/hijack.go

```diff
@@ -206,10 +206,3 @@ type hijackedConn struct {
 func (c *hijackedConn) Read(b []byte) (int, error) {
 	return c.r.Read(b)
 }
-
-func (c *hijackedConn) CloseWrite() error {
```

Comment:
this looks off

Reply:
Again, agree, don't know why the bump picked up the files in github.com or in k8s.io